src/modules/facility-loan-transaction/facility-loan-transaction.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from '@nestjs/common';
+import { AcbsModule } from '@ukef/modules/acbs/acbs.module';
+
+import { DateModule } from '../date/date.module';
```

Comment:
Should this be part of `@ukef/module...`?

Reply:
Thanks, changed.